docker/README.md

```diff
@@ -0,0 +1,108 @@
+<!---
+Licensed under Creative Commons Attribution 4.0 International License
+https://creativecommons.org/licenses/by/4.0/
+--->
+
+# Docker Tools and Usages #
+
+This directory contains scripts useful for building and running a
+container with the PDO contracts. For the most part, this directory
+could be copied to any host (even without PDO otherwise installed) to
+build, configure, and execute PDO contracts code.
```

Comment:
Hmm, this is true only _after having run `make repository` from a full checkout_?  Otherwise you are missing stuff  in the generated `repository sub-directory` like the get-version scripts and a alike? In which case it's virtually the same as just checking out from scratch size-wise?

Reply:
will clarify. but you do not have to have the repository directory to run pre-built images. you do need the xfer directory, however.

Reply:
Oops, forgot that we actually fixed this in pdo for `get-version` and you replicated the same pattern here. But to make it really work you will have then to manually specify `PDO_VERSION` and `CONTRACTS_VERSION` when invoking `make` (or tag the desired images with `latest`)